modules/apps/frontend-css/frontend-css-variables-web/src/test/resources/com/liferay/frontend/css/variables/web/internal/servlet/taglib/liferay_css_variables_1.html

```diff
@@ -0,0 +1,5 @@
+<style data-senna-track="temporary" id="liferayCssVariablesCSS" type="text/css">
```

Comment:
@izaera, I think we need to remove the `id` from the fixtures to make the tests pass again, right?

Reply:
Yes. I forgot to run the tests after the refactor :-(.